lib/govuk_index/presenters/indexable_content_presenter.rb

```diff
@@ -0,0 +1,31 @@
+module GovukIndex
+  class IndexableContentPresenter
+    def initialize(details)
+      @details = details
+      @sanitiser = IndexableContentSanitiser.new
+    end
+
+    def indexable_content
+      return nil if details.nil?
+      sanitiser.clean(elements)
+    end
+
+  private
+
+    attr_reader :details, :sanitiser
+
+    def elements
+      if details['parts']
+        parts(details)
```

Comment:
Just to clarify: if a guide has parts, the indexable content just comes from the parts? We don't include any details from the top-level page?